movie_library/repositories/movie_repository.py

```diff
@@ -0,0 +1,43 @@
+
+from ..models.movie import Movie
+from .base_repository import BaseRepository
+
+class MovieRepository():
+    """
+    this is used to perform the ORM operation and all database operations
+    """
+
+    '''def get_base_query(self, filters=None, **options):
+        return Movie.objects.all()
+
+    def get_movies(self, filters=None):
+        query = self.get_base_query(filters)
+        return query.all()'''
+
+    def get_movies(self, filters=None):
+        query = BaseRepository().get_base_query(Movie, filters)
+        return query.all()
+
+    def create_movie(self, **data):
+
+        movie = Movie(**data)
+        movie.save()
+        return movie
+
+    def update_movie(self, movie , **data):
```

Comment:
you have the data type is dictionary and you can simply pass it as data 
since the most common use to pass extra argument to a function is using  *args, **kwarg